pumps/kafka.go

```diff
@@ -90,6 +91,10 @@ func (k *KafkaPump) Init(config interface{}) error {
 	}
 
 	processPumpEnvVars(k, k.log, k.kafkaConf, kafkaDefaultENV)
+	// This interface field is not reached by envconfig library, that's why we manually check it
+	if os.Getenv("TYK_PMP_PUMPS_KAFKA_META_TIMEOUT") != "" {
```

Comment:
I'm not too fond of this way of getting the env var, but `envconfig` library isn't getting the env var if the data type is an `interface`